cmd/root.go

```diff
@@ -184,7 +184,7 @@ func CreateSystem() *core.System {
 	cryptoInstance := crypto.NewCryptoInstance()
 	jsonld := jsonld.NewJSONLDInstance()
 	storageInstance := storage.New()
-	didStore := store.NewBBoltStore()
+	didStore := store.NewStore(storageInstance.GetProvider(vdr.ModuleName))
```

Comment:
moduleName is upper case. This is translated to a directory. This will work on your mac but will be not backwards compatible on linux.

Reply:
module name is convert to lowercase by storage engine